Assets/Scripts/Models/Character/Character.cs

```diff
@@ -818,8 +799,8 @@ private void Update_DoMovement(float deltaTime)
         // But when we do the pathfinding system, we'll likely
         // switch to something like Manhattan or Chebyshev distance
         float distToTravel = Mathf.Sqrt(
-            Mathf.Pow(CurrTile.X - nextTile.X, 2) +
-            Mathf.Pow(CurrTile.Y - nextTile.Y, 2));
+                                 Mathf.Pow(CurrTile.X - nextTile.X, 2) +
```

Comment:
MonoDevelop is adding them. It's also correcting white space all over the file. I wonder why it isn't changing it to this for you. I have set the settings to what is in the contribution guide.